test/TBD/implied_objc_symbols.swift

```diff
@@ -0,0 +1,11 @@
+// REQUIRES: VENDOR=apple
+// RUN: %empty-directory(%t)
+
+// RUN: echo "import Foundation" > %t/main.swift
+// RUN: echo "@objc public class ObjCOnly {}" >> %t/main.swift
```

Comment:
can you add a check for existence of the symbol name too, not just that the prefix doesn't appear?

Reply:
Done

Reply:
thanks!